pkg/sql/schemachanger/corpus/BUILD.bazel

```diff
@@ -18,4 +18,17 @@ go_library(
     ],
 )
 
+go_test(
+    name = "corpus_test",
+    srcs = ["corpus_test.go"],
```

Comment:
I think you should add `tags = ["integration"],` to this `go_test` definition. Otherwise it'll be run as part of the unit test job which I think is not what you want.